docscan/constants/constants.go

```diff
@@ -7,9 +7,11 @@ const (
 	IDDocumentTextDataExtraction            string = "ID_DOCUMENT_TEXT_DATA_EXTRACTION"
 	IDDocumentFaceMatch                     string = "ID_DOCUMENT_FACE_MATCH"
 	SupplementaryDocumentTextDataCheck      string = "SUPPLEMENTARY_DOCUMENT_TEXT_DATA_CHECK"
+	ThirdPartyIdentityCheck                 string = "THIRD_PARTY_IDENTITY"
 	SupplementaryDocumentTextDataExtraction string = "SUPPLEMENTARY_DOCUMENT_TEXT_DATA_EXTRACTION"
 	Liveness                                string = "LIVENESS"
 	Zoom                                    string = "ZOOM"
+	SupplementaryDocumentType               string = "SUPPLEMENTARY_DOCUMENT"
```

Comment:
Doesn't look like this is used

Reply:
It's used downstream in the sandbox repo.